libs/shared/contentful/src/lib/__mocks__/contentful.client.ts

```diff
@@ -1,6 +1,9 @@
-export const mockQuery = jest
-  .fn()
-  .mockResolvedValue({ data: { purchaseCollection: { items: [] } } });
+export const mockQuery = jest.fn().mockResolvedValue({
+  data: {
+    purchaseCollection: { items: [] },
+    serviceCollection: { items: [] },
+  },
+});
```

Comment:
This should probably be removed, since now it's returning invalid data and only possibly applied to `purchaseCollection` in the first place.